.github/workflows/label_new_pr.yml

```
@@ -25,3 +25,8 @@ jobs:
           client_payload: ${{ format('{{"pr_url":"{0}"}}', env.PR_URL) }}
           wait_time: 5 # check every 5 seconds
           max_time: 120 # timeout after 2 minutes
+      - uses: actions/checkout@v2
+      - uses: actions-ecosystem/action-add-labels@v1
+        if: ${{failure()}}
+        with:
+          labels: "🚦 status: awaiting triage"
```

Adding some spaces will help a bit with readability.
```suggestion
          max_time: 120 # timeout after 2 minutes
          
      - uses: actions/checkout@v2
      - name: Add default label in case of failure
        uses: actions-ecosystem/action-add-labels@v1
        if: ${{ failure() }}
        with:
          labels: "🚦 status: awaiting triage"
```